### PR TITLE
Add Post Scriptum

### PR DESCRIPTION
**Pull request by southnode**
Based on the standard Squad Dedicated Server template recently complete - PS is effectively a carbon copy game by the same publisher, so shares a lot of the same things.

### CODE REVIEW

**Comment by Greelan** [comment]
This should really use the new ports system

**Comment by southnode** [comment]
@Greelan agreed but I have zero idea how to implement that.

**Comment by Greelan** [comment]
Plenty of other templates to show you how xD

File: post-scriptum.kvp [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+Console.UserJoinRegex=
+Console.UserLeaveRegex=
+Meta.Author=Southnode
+Meta.ConfigManifest=post-scriptum-config.json
+Meta.MetaConfigManifest=post-scriptum-metaconfig.json
+Meta.ConfigRoot=post-scriptum.kvp
+Meta.Description=Post Scriptum Dedicated Server
+Meta.DisplayImageSource=steam:736220
+Meta.DisplayName=Post Scriptum
+Meta.EndpointURIFormat=steam://connect/{ip}:{GenericModule.App.ApplicationPort2}
```
Comment (Greelan) [inline post-scriptum.kvp:42]: Should be ... App.Ports.$QueryPort

File: post-scriptum.kvp [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+App.ExitMethod=OS_CLOSE
+App.ExitString=stop
+App.RCONConnectDelaySeconds=15
+App.RCONConnectRetrySeconds=45
+App.HasReadableConsole=true
+App.HasWritableConsole=true
+App.RapidStartup=false
+App.RemoteAdminPort=21214
+App.RootDir=./post-scriptum-server/
+App.UpdateSource=Multi
```
Comment (Greelan) [inline post-scriptum.kvp:25]: Not required

File: post-scriptum.kvp [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+App.ExecutableLinux=746200/PostScriptum/Binaries/Linux/PostScriptumServer
+App.ExecutableWin=746200\PostScriptum\Binaries\Win64\PostScriptumServer.exe
+App.ExitMethod=OS_CLOSE
+App.ExitString=stop
+App.RCONConnectDelaySeconds=15
+App.RCONConnectRetrySeconds=45
+App.HasReadableConsole=true
+App.HasWritableConsole=true
+App.RapidStartup=false
+App.RemoteAdminPort=21214
```
Comment (Greelan) [inline post-scriptum.kvp:23]: Redundant

File: post-scriptum.kvp (hunk 2 above)
Comment (southnode) [reply post-scriptum.kvp:25]: Both of them?

Comment (Greelan) [reply post-scriptum.kvp:25]: Both of what?

Comment (southnode) [reply post-scriptum.kvp]: Change implemented

File: post-scriptum.kvp (hunk 3 above)
Comment (southnode) [reply post-scriptum.kvp]: Change implemented

**Comment by Greelan** [comment]
Have suggested some tweaks to the kvp.

Do the RCONConnectDelay and RCONRetryDelay settings really need to be so long?

Does the metaconfig successfully write to the file, even without ConfigFormatRegex specified?

I will have a closer look at the manifest later.

**Comment by southnode** [comment]
I don't think it's ever had them honestly.

Metaconfig has "ConfigFormat": "{0}=\"{1}\"" in it so one would imagine it would work unless I need to put that in multiple spots...

**Comment by Greelan** [comment]
> I don't think it's ever had them honestly.
> 
> Metaconfig has "ConfigFormat": "{0}="{1}"" in it so one would imagine it would work unless I need to put that in multiple spots...

Actually tested it?

**Comment by southnode** [comment]
Yes this has been tested and does change the Server.cfg values - for example the ServerName value

**Comment by southnode** [comment]
RCONConnectDelay and RCONRetryDelay have been adjusted. OWI UE4 games such as Squad and Post Scriptum are particularly susceptible to RCON issues (it's a flimsy system that falls over if you spam it) so these are relatively safe numbers.

**Review by IceOfWraith — APPROVED** [review]
Very good! Thanks for the addition!

**Review by IceOfWraith — APPROVED** [review]
Sorry about that and thank you!

**Pull request merged**